src/controllers/project.js

```diff
@@ -0,0 +1,21 @@
+const Project = require('../models/project');
+
+const filterProjects = async (req, res) => {
```

Comment:
okay, this function name is not correct I guess, it says filterProjects, but it's using the query to search within the projects using the regex, so maybe the function name can be better.